htcanalyze/htcanalyze.py

```diff
@@ -72,34 +72,35 @@ def manage_thresholds(self, resources: dict) -> dict:
         """
         # change to list, to avoid numpy type errors
         resources.update(Usage=list(resources["Usage"]))
+        # TODO: What does i do? Making resources['Requested'] a dict would be more descriptive
         for i, _ in enumerate(resources['Resources']):
```

Comment:
I seem to have missed this in an earlier PR, but this is wrong and not what my comments suggested you to do.

First and foremost, to address @jonamfischer  comment: `i` is an iterator that you get from `enumerate`. It basically counts from 0..n, where n is the number of objects in the iterable (`resources['Resources']`).

On to a comment for @mathisloevenich: You assign the contents of `resources['Resources']` to `_`, which is commonly used as a variable that remains unused. Why do you do this?

Take, for example, `for i, resource in enumerate(resources['Resources']):`. This makes `resource['Resources'][i]` and 'resource' functionally equivalent, except now you never have to worry if you ever decide to change the iterator inside the loop. 

Right now you're using enumerate, but you're throwing away the reason fr using it in the first place.

Since you're looking at the `Requested`, `Usage`, etc., my suggestion was to refactor the resources dictionary, so that you'd get `resources = [{'requested': 1, 'usage': 2}, {'requested': 'a', 'usage': 'b'}]`, instead of what you currently have `resources = {'requested': [1, 'a'], 'usage': [2, 'b']}. That way, the structure is much more relevant to how you are using it, and you don't have to retrieve values hoping that their list order is the same.